app/Traits/AccessControlAPI.php

```diff
@@ -140,16 +140,18 @@ function () use ($api_key, $fileset_hash, $access_group_ids) {
 
                 $dbp_database = config('database.connections.dbp.database');
 
+                $is_bileis_api_key =  $this->isBibleIsApiKey($api_key);
```

Comment:
spelling error

Reply:
@bradflood I have changed it. It is named: `doesApiKeyBelongToBibleis`